pkgs/applications/networking/mailreaders/mailpile/default.nix

```diff
@@ -2,12 +2,12 @@
 
 python2Packages.buildPythonApplication rec {
   name = "mailpile-${version}";
-  version = "0.4.1";
+  version = "0.5.2";
 
   src = fetchgit {
```

Comment:
hmm, I moved to fetchFromGitHub but it still building, I'll try to verify later if it's working